lib/multicell_location/services/CMakeLists.txt

```diff
@@ -10,4 +10,3 @@ zephyr_library_sources(location_service.c)
 
 zephyr_library_sources_ifdef(CONFIG_MULTICELL_LOCATION_SERVICE_NRF_CLOUD nrf_cloud_integration.c)
 zephyr_library_sources_ifdef(CONFIG_MULTICELL_LOCATION_SERVICE_HERE here_integration.c)
-zephyr_library_sources_ifdef(CONFIG_MULTICELL_LOCATION_SERVICE_POLTE polte_integration.c)
```

Comment:
Looks like files `polte_integration.c` and `polte_integration.h` have not been removed in the commit?

Reply:
Yep, forgot that. Done.